pressbooks/pressbooks-book-directory-fe
# Fixed pagination reset

Related Issue #176 

This PR reset the pagination when the search query changes.

### How to test

1. Perform a search
2. Navigate to any of the pages
3. Perform another search or clear the input
4. Pagination should be reset
